storybook/cypress/e2e/header.cy.ts

```diff
@@ -56,11 +56,21 @@ describe("Header", () => {
   }
 
   it("should be accessible", () => {
+    prepareComponent();
+
     cy.checkA11y("dso-header");
-    // cy.percySnapshot()
+
+    cy.get("dso-header.hydrated")
+      .get("@dsoHeaderShadow")
+      .get("dso-icon.hydrated")
+      .should("exist")
```

Comment:
Deze gets doen niets op zichzelf: Volgens mij willen we dit:

```suggestion
    cy.get("dso-header.hydrated, dso-icon.hydrated")
      .should("exist")
      .get("@dsoHeaderShadow")
      .should("exist")
```

Reply:
Gedaan.

Reply:
Er wordt nu nog maar 1 screenshot gemaakt in de test `should be accessible`. Ik maak een issue aan om het visueel regressie testen van het header-component.